src/Microsoft.Security.Utilities.Core/Releases.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Security.Utilities;
+
+/// <summary>
+/// This class contains the version numbers of the library releases. Prior to
+/// version 1.4.15, we did not properly maintain change tracking in our release
+/// history and we have not back-filled release notes for all code changes. We
+/// have added an explicit release version for every historical version where a
+/// rule was introduced. And so we have release here for version 1.4.11 (because
+/// SEC101/AadClientAppLegacyCredentials was added in this version) but we do
+/// not have an entry for 1.4.14 (because no new rules were added in this
+/// version).
+/// </summary>
+internal static class Releases
+{
+    // Current
+    public static Version Unreleased => new Version(1, 18, 0);
```

Comment:
What is the reason for not calling this `Version_1_18_0` straight away? I'm worried that we'll forget to do that at release time.

Reply:
The reason I had in mind is that the specific version implied by `Unreleased` isn't really known until we finish all the work. e.g., `1.18` could turn into `2.0` depending on what happens in the next check-in.

As we prepare for release, what we'd do is perform an IDE rename of `Unreleased` to the final version and then put in a new prospective `Unreleased` value.

I thought this was easiest. But maybe it's more theater than we need. If we take your suggestion, we just use `1_18` as long as it's relevant and then rename it on accepting an ureleased major breaking change. OK, so I'll take your note. :)